src/ui/components/cards/cardcomponent.js

```diff
@@ -64,11 +67,52 @@ export default class CardComponent extends Component {
     this.verticalKey = data.verticalKey;
   }
 
+  onMount () {
+    const rtfElement = DOM.query(this._container, '.js-yxt-rtfValue');
+    rtfElement && this._handleClickAnalytics(rtfElement);
+  }
+
+  /**
+   * Creates a handler for the click analytics of a Rich Text attribute.
+   *
+   * @param {Node} rtfElement The Rich Text attribute.
+   */
+  _handleClickAnalytics (rtfElement) {
+    const self = this;
+    const eventHandler = event => {
+      if (!event.target.hasAttribute('data-cta-type')) {
+        return;
+      }
+      const ctaType = event.target.getAttribute('data-cta-type');
```

Comment:
could we do something like `const { ctaType, fieldName } = rtfElement.dataset` here?